src/MysqldumpGdpr.php

```diff
@@ -35,6 +35,11 @@ public function __construct(
             unset($dumpSettings['gdpr-replacements']);
         }
 
+        if (array_key_exists('gdpr-replacements-file', $dumpSettings)) {
```

Comment:
You're actually doing double work for yourself here - since you've already read and have passed in the data via the $dumpsettings variable. So I'd say, let's remove this as well.